src/test/java/com/odysseusinc/arachne/portal/api/v1/controller/AnalysisControllerTests.java

```diff
@@ -264,13 +264,11 @@ public void testDeleteAnalysis() throws Exception {
     public void testUploadCodeFile() throws Exception {
 
         FileInputStream fileInputStream = new FileInputStream(this.getClass().getResource("/test.jpg").getPath());
-        MockMultipartFile multipartFile = new MockMultipartFile("file", "test.jpg", "image/jpeg", fileInputStream);
-
+        MockMultipartFile multipartFile = new MockMultipartFile("files", "test.jpg", "application/octet-stream", fileInputStream);
         this.mvc.perform(
```

Comment:
Why MIME has changed?

Reply:
I repeated it from frontend part but this change has no effect so will roll it back